character.h

```diff
@@ -0,0 +1,17 @@
+#ifndef CHARACTER_H
+#define CHARACTER_H
+#include <iostream>
+#include <string>
+
+class Character {
+    const std::string Name;
+    int HP;
+    int DMG;
```

Comment:
A DMG is legyen const: egyelőre a specifikáció szerint nem változhat, ezért jobb, ha a fordító sem engedi meg, hogy valami véletlenül megváltoztassa.